locales/en-US/server.ftl

```diff
@@ -217,7 +220,12 @@ shotIndexPagePreviousPage
     .title = Previous page
 shotIndexPageNextPage
     .title = Next page
-
+// This symbol is used in the lower right corner of the card for a shot on the
+// My Shots page to indicate that the shot does not expire. It should be a
+// single character (or simply nothing if no such symbol is available for a
```

Comment:
I wonder if we risk breaking something without a value. Maybe suggest `or a character like - if…`

Reply:
Nothing should break if we don't have a value.  That's why I've left out the default content at https://github.com/mozilla-services/screenshots/pull/3965/files#diff-d45944c551eead471a128c9a716e2a79R309

Reply:
OK, then feel free to keep the existing comment